lib/Test2/API.pm

```diff
@@ -110,7 +110,7 @@ use base 'Exporter';
 # This will make sure that Context.pm is loaded the first time this module is
 # imported, then the regular import method is swapped into place.
 sub import {
-    require Test2::API::Context unless $INC{'Test2/API/Context.pm'};
```

Comment:
schwern had benchmarks that showed that  ```require foo unless $INC{'foo.pm'}``` was significantly faster than letting require do the checking.